File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/groups/GroupsMvpView.java
package ru.ulstu_team.ulstuschedule.ui.groups;

import java.util.List;

import ru.ulstu_team.ulstuschedule.data.model.Group;
import ru.ulstu_team.ulstuschedule.ui.base.MvpView;

public interface GroupsMvpView extends MvpView {

    void showGroups(List<Group> groups);

    void showEmptyList();

    void showError();

}

File: app/src/main/java/ru/ulstu_team/ulstuschedule/ui/old/groups/GroupsMvpView.kt
package ru.ulstu_team.ulstuschedule.ui.old.groups

import ru.ulstu_team.ulstuschedule.data.model.Group
import ru.ulstu_team.ulstuschedule.ui.base.MvpView

interface GroupsMvpView : MvpView {

    fun showGroups(groups: List<Group>)

    fun showEmptyList()

    fun showError()

}
